<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DictionaryForAll</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="17" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入要查询的单词</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="23" column="3">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="18" column="2" rowspan="2" colspan="2">
     <widget class="QListWidget" name="listWidget_2">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="8">
     <widget class="QCheckBox" name="checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>显示图片</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="17" column="10">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="17" column="9">
     <widget class="QPushButton" name="pushButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="25" column="0" colspan="4">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" rowspan="2" colspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="8" colspan="3">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
    <item row="17" column="6" colspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="17" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="18" column="6" colspan="5">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="24" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="19" column="6" rowspan="8" colspan="5" alignment="Qt::AlignTop">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="21" column="3">
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>Translate</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0" rowspan="4" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="22" column="3">
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <family>等线</family>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="font">
     <font>
      <family>等线</family>
     </font>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionSet_apis"/>
    <addaction name="actionPic_sources"/>
    <addaction name="actionFunctions"/>
   </widget>
   <widget class="QMenu" name="menuLanguages">
    <property name="font">
     <font>
      <family>等线</family>
     </font>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuLanguages"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSet_apis">
   <property name="text">
    <string>接口设置</string>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>简体中文</string>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
  </action>
  <action name="actionPic_sources">
   <property name="text">
    <string>图源设置</string>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
  </action>
  <action name="actionFunctions">
   <property name="text">
    <string>功能设置</string>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
